golden_data.Rmd

```diff
@@ -71,7 +71,9 @@ You can export this table similar to before.
 Click the ![](images/uniprot_dl_icon.png){width=50} button to select all sequences, change the format to "Tab-separated", then click "Go" to save the table to your computer.
 Change the file name to `UniProt_PmoA.tab`.
 
-To gather all sequences in UniProt for the XmoA reference package, repeat the previous two steps with the string "Ammonia monooxygenase alpha subunit" and gene name "amoA".
+To gather all sequences in UniProt for the XmoA reference package, repeat the previous three steps with the string "Ammonia monooxygenase alpha subunit", *Gene name [GN]* "amoA", and the same taxonomic filter.
+There should be >3,000 entries returned.
```

Comment:
Ah, I thought amoA was an archean gene so I switched the taxonomic filter. oops, my bad.